src/dataset.ts

```diff
@@ -119,6 +119,11 @@ export interface Structure {
      * expressed in Angströms.
      */
     cell?: number[];
+    /**
+     * bonds in structures given as `[id1 id2 order]`, where
+     * **id1**, **id2**, and **order** are integers.
```

Comment:
```suggestion
     * bonds in structures given as `[i, j, order]`, where
     * `i`, `j`, and `order` are integers. `i` and `j` are the indices of 
     * the atoms bonded together, and `order` is the bond order, which can be 
     * 1 (for single bonds) or 2 (for double bonds)
```

are other bond order supported?

Reply:
I wonder if we should be more general, because bond orders could also be 1.5 for aromatic, and integers up to 9 for rdkit molecules. I will use definitions in 3dmol

Reply:
Note, that this should be handled by the user/structure definition functions to convert appropriately.

Reply:
yes, but we need to have a fairly clear definition of what things mean =). For example I don't think that all kinds of bond order in rdkit are worth supporting, since they will not have a separate representation in 3Dmol (or any other molecule viewer we'll use at some point)

Reply:
Agreed, so I have used your suggestion (adding 3 for triple bonds), and added a converter in `_stk.py` specific to what I know stk does.